src/pybind/rados/mock_rados.pxi

```diff
@@ -331,6 +331,13 @@ cdef nogil:
         pass
     int rados_unlock(rados_ioctx_t io, const char * o, const char * name, const char * cookie):
         pass
+    ssize_t rados_list_lockers(rados_ioctx_t io, const char *o, const char *name, int *exclusive,
+                               char *tag, size_t *tag_len, char *clients, size_t *clients_len,
+                               char *cookies, size_t *cookies_len, char *addrs, size_t *addrs_len):
+        pass
+    int rados_break_lock(rados_ioctx_t io, const char *o, const char *name, const char *client,
+                         const char *cookie):
+        pass
```

Comment:
Nit: Don't make the lines too long unnecessarily and keep the order the same as in `c_rados.pxd` and `rados.pyx` files:
```suggestion
    int rados_break_lock(rados_ioctx_t io, const char *o, const char *name,
                         const char *client, const char *cookie):
        pass
    ssize_t rados_list_lockers(rados_ioctx_t io, const char *o,
                               const char *name, int *exclusive,
                               char *tag, size_t *tag_len,
                               char *clients, size_t *clients_len,
                               char *cookies, size_t *cookies_len,
                               char *addrs, size_t *addrs_len):
        pass
```